votor/src/event_handler.rs

```diff
@@ -606,6 +617,8 @@ impl EventHandler {
         )?;
         pending_blocks.remove(&slot);
 
+        Self::try_final(my_pubkey, (slot, block_id), voting_context, votes)?;
```

Comment:
nice catch!

Reply:
yes, without this we would have never been able to slow finalize!